build-tools/src/main/kotlin/org/jetbrains/kotlin/Utils.kt

```diff
@@ -54,6 +54,12 @@ val Project.globalTestArgs: List<String>
 fun Project.platformManager() = findProperty("platformManager") as PlatformManager
 fun Project.testTarget() = findProperty("target") as KonanTarget
 
+fun KonanTarget.supportsCodeCoverage(): Boolean =
+        this == KonanTarget.MINGW_X64 ||
```

Comment:
Duplication with CoverageManager.kt? Shalln't be in shared?

Reply:
I wasn't sure that there is place to share such logic. But now I remembered `shared` package.